OCR/frontend/package.json

```diff
@@ -8,14 +8,16 @@
     "build": "tsc -b && vite build",
     "lint": "eslint .",
     "preview": "vite preview",
-    "test": "vitest",
+    "test": "vitest run --coverage.all",
```

Comment:
I would recommend separating this to a different command, we can do that in a future PR